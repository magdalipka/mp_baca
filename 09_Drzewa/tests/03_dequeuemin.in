1
14
ENQUE 35 Andrzej Wolny
ENQUE 30 Andrzej Inny
ENQUE 40 Iwona Wolny
ENQUE 45 Iwona Inny
DEQUEMIN
INORDER
DEQUEMIN
INORDER
DEQUEMIN
INORDER
DEQUEMIN
INORDER
DEQUEMIN
INORDER
