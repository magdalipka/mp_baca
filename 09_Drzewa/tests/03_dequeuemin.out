ZESTAW 1
DEQUEMIN: 30 - Andrzej Inny
INORDER: 35 - Andrzej Wolny, 40 - Iwona Wolny, 45 - Iwona Inny
DEQUEMIN: 35 - Andrzej Wolny
INORDER: 40 - Iwona Wolny, 45 - Iwona Inny
DEQUEMIN: 40 - Iwona Wolny
INORDER: 45 - Iwona Inny
DEQUEMIN: 45 - Iwona Inny
INORDER: BRAK
DEQUEMIN: BRAK
INORDER: BRAK